### PR TITLE
feat: add extensionList to quote request/response

### PR DESCRIPTION
**Pull request by shashi165**
Fixes: https://github.com/mojaloop/project/issues/2116

### CODE REVIEW

File: src/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/sdk-scheme-adapter",
-  "version": "11.15.0",
+  "version": "11.16.0",
```
Comment (eoln) [inline src/package.json:3]: there is no need to manually bump version -> it is done automatically via CI/CD

**Review by eoln — CHANGES_REQUESTED** [review]

**Review by mdebarros — APPROVED** [review]
LGTM!

Except for @eoln's comment.

**Review by mdebarros — APPROVED** [review]
+1

File: src/package.json (hunk 1 above)
Comment (shashi165) [reply src/package.json]: @eoln - Fixed, please check now.

**Review by eoln — APPROVED** [review]

**Pull request merged**